apps/api/src/migration/1676577091787-RemoveOldUnusedTables.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class RemoveOldUnusedTables1676577091787 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      DROP TABLE IF EXISTS form;
+      DROP TABLE IF EXISTS submission;
+      DROP TABLE IF EXISTS applicant_status;
+      DROP TABLE IF EXISTS applicants;
+      DROP TABLE IF EXISTS applicant_status_audit;
+      DROP TABLE IF EXISTS applicant_audit;
+    `);
+  }
+
+  public async down(): Promise<void> {
+    // no rollback with this migration
```

Comment:
Do we have a migration with these initial tables? If so, it would be a nice to have to be able to add them back in. 
Totally a nitpick though.

Reply:
I can't find any reference to these tables in the project